src/main/java/fact/Utils.java

```diff
@@ -522,4 +523,17 @@ public static int absPos(int pix, int slice, int roi) {
 		return pixelStatistics;
 	}
 
+
+	/**
+	 * Flatten a 2d array
+     *
+     * @param array2d 2dim double array
+     * @return flattend double array
+	 */
+	public static double[] flatten2dArray(double[][] array2d) {
```

Comment:
No sure we need an extra function for this.  The one line you wrote here is pretty nice and already right?  Maybe it can be used directly.